demos/common/python/openvino/model_zoo/model_api/README.md

```diff
@@ -0,0 +1,145 @@
+# Python* Model API package
+
+Model API package is a set of wrapper classes for particular tasks and model architectures, simplifying data preprocess and postprocess as well as routine procedures (model loading, asynchronous execution, etc...)
+Model API wrappers hide the specific code inside and work as black-box: application feeds model class with input data, then the model returns post-processed output data in user-friendly format.
+
+## Package structure
+
+The Python* Model API consists of 3 libraries:
+* _adapters_ - implement a common interface to allow Model API wrappers usage with different executors: OpenVINO, ONNX, etc. See [Model API adapters](#model-api-adapters) section
+* _models_ - implement wrappers for each architecture. See [Model API Wrappers](#model-api-wrappers) section
+* _pipelines_ - implement pipelines for model inference and manage the synchronous/asynchronous execution. See [Model API Pipelines](#model-api-pipelines) section
+
+## Building Python* Model API package
+For installation Python (version 3.6 or higher) is required. The installation is available from source.
+
+Use the following command to install Python* Model API from source:
+```sh
+pip install <omz_dir>/demos/common/python
+```
+
+Alternatively, you can generate the package using a wheel. Follow the steps below:
+1. Build the wheel.
+
+```sh
+python <omz_dir>/demos/common/python/setup.py bdist_wheel
+```
+The wheel should appear in the dist folder.
+Name example: `openmodelzoo_modelapi-0.0.0-py3-none-any.whl`
+
+2. Install the package in the clean environment with `--force-reinstall` key.
+```sh
+pip install openmodelzoo_modelapi-0.0.0-py3-none-any.whl --force-reinstall
+```
+
+To verify the package is installed, you might use the following command:
+```sh
+python -c "from openvino.model_zoo import model_api"
+```
+
+> **NOTE**: On Linux and macOS, you may need to type `python3` instead of `python`. You may also need to [install pip](https://pip.pypa.io/en/stable/installation/).
```

Comment:
probably it is extra info, it should be obvious that user should use appropriate python version, anyway it is not cover all cases (e.g. working in virual environment does not requered to use python3 / pip3 tools if you do not create environment with python2

Reply:
Removed the note